package.json

```diff
@@ -5,9 +5,10 @@
   "author": "Discourse",
   "license": "MIT",
   "devDependencies": {
-    "eslint-config-discourse": "latest",
-    "eslint-plugin-simple-import-sort": "^10.0.0",
-    "prettier": "^3.0.3",
-    "prettier-plugin-ember-template-tag": "^1.1.0"
+    "@discourse/lint-configs": "^1.0.0",
+    "ember-template-lint": "^5.11.2",
+    "eslint": "^8.51.0",
+    "minimasonry": "^1.3.2",
```

Comment:
sorry, this one is actually a runtime dependency of discourse-gifs, not related to linting 😅 

```suggestion
```

but you will need the updates to eslint, prettier, and template-lint configs from that PR 🙂

Reply:
being early adopter ftw! 😁